src/Contrib/StackSdks/Masa.Contrib.StackSdks.Auth/Service/UserService.cs

```diff
@@ -165,8 +171,8 @@ public async Task<bool> DisableUserAsync(DisableUserModel user)
 
     public async Task<List<UserSimpleModel>> GetListByAccountAsync(IEnumerable<string> accounts)
     {
-        var requestUri = $"api/user/GetListByAccount";
```

Comment:
in line 157 `Possible null reference argument for parameter 'json' in 'T? JsonSerializer.Deserialize<T>(string json, JsonSerializerOptions? options = null)'.`